.github/workflows/pull-request-build-site.yml

```diff
@@ -1,22 +1,26 @@
-name: CI
+# This GitHub Action runs in each pull-request.
+# It tests building of the static Jekyll site.
+
+name: Pull-request - Build static site
 
 on:
-  push:
-    branches: ["main"]
```

Comment:
I removed
```
push:
    branches: ["main"]
```
because the GitHub action that deploys the `main` branch to GitHub Pages (see [/.github/workflows/pages.yml](https://github.com/CambridgeTechCollective/CambridgeCollection/blob/main/.github/workflows/pages.yml)) indirectly tests building of the Jekyll site.